src/test/java/cz/cvut/kbss/termit/rest/ConfigurationControllerSecurityTest.java

```diff
@@ -65,6 +65,7 @@ private static ConfigurationDto generateConfiguration() {
 
     @Test
     void getConfigurationReturnsConfigurationWithoutRolesWhenUserIsNotAuthenticated() throws Exception {
+        Environment.resetCurrentUser();
```

Comment:
IMHO not a good fix, see https://github.com/kbss-cvut/termit/pull/115/commits/76e299d85f5e0c79fa2e766db5092c9c48a42030